include/openthread/link_raw.h

```diff
@@ -354,6 +354,15 @@ ThreadError otLinkRawSrcMatchClearShortEntries(otInstance *aInstance);
 ThreadError otLinkRawSrcMatchClearExtEntries(otInstance *aInstance);
 
 /**
+ * Get the radio receive sensitivity value.
+ *
+ * @param[in] aInstance  The OpenThread instance structure.
+ *
+ * @returns The radio receive sensitivity value in dBm.
+ */
+int8_t otLinkRawGetReceiveSensitivity(otInstance *aInstance);
```

Comment:
Suggestion: We can add it to `link.h` instead as `otLinkGetReceiveSensetivity` so it's not tied to only link-raw case.